### PR TITLE
MDJ Table Tooltip

### PR DESCRIPTION
**Pull request by marioaks**
Adds correct tooltip to the MDJ Table.
Adds some very minor style updates.

### CODE REVIEW

File: src/css/visualization.css
Comment (alexwgraves) [inline src/css/visualization.css]: Remove commented-out stuff

File: src/css/visualization.css
Comment (alexwgraves) [inline src/css/visualization.css]: Remove extra return

**Review by alexwgraves — CHANGES_REQUESTED** [review]
Small styles changes

**Comment by alexwgraves** [comment]
In the future please wait for an approving review before merging!

**Comment by marioaks** [comment]
Oh I thought it was approved, my bad. Made those small style changes

**Pull request merged**
